activiti-core/activiti-api-impl/activiti-api-process-model-impl/src/main/java/org/activiti/api/runtime/model/impl/ObjectValueToStringConverter.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2010-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.api.runtime.model.impl;
+
+import java.util.Map;
+
+import org.springframework.core.convert.converter.Converter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@ProcessVariableTypeConverter
+public class ObjectValueToStringConverter implements Converter<ObjectValue, String> {
+    private final ObjectMapper objectMapper;
+
+    public ObjectValueToStringConverter(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    @Override
+    public String convert(ObjectValue source) {
+
+        try {
+            Map<String, Object> value = objectMapper.convertValue(source, Map.class);
+
+            if (Map.class.isInstance(value.get("object"))) {
```

Comment:
It's probably better to extract a constant for `"object"`.

Reply:
Done